src/Robot.cpp

```diff
@@ -67,8 +70,8 @@ void Robot::TeleopInit() {
 	if (autonomousCommand.get() != nullptr)
 		autonomousCommand->Cancel();
 	//Setting starting values of Pneumatics......................
-	RobotMap::chassisshiftLeft->Set(DoubleSolenoid::kReverse);
-	RobotMap::chassisshiftRight->Set(DoubleSolenoid::kReverse);
+	RobotMap::chassisshift->Set(DoubleSolenoid::kReverse);
```

Comment:
Yes, pneumatics are always just split if there are two. For example, same for the shooter lift.